code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -259,6 +260,28 @@ GLOBAL_LIST_INIT(bananium_recipes, list ( \
 	recipes = GLOB.bananium_recipes
 	. = ..()
 
+/*
+ * Mime
+ */
+/obj/item/stack/sheet/mineral/tranquillite
+	name = "tranquillite"
+	icon_state = "sheet-tranquillite"
+	item_state = "sheet-tranquillite"
+	singular_name = "tranquillite sheet"
```

Comment:
call them berets like on Para, not sheets

Reply:
alright, i'll do that.